src/Symfony/Bundle/FrameworkBundle/Test/KernelTestCase.php

```diff
@@ -106,16 +106,17 @@ private static function getPhpUnitCliConfigArgument()
      */
     protected static function getKernelClass()
     {
-        if (isset($_SERVER['KERNEL_CLASS'])) {
-            if (!class_exists($class = $_SERVER['KERNEL_CLASS'])) {
+        if (isset($_SERVER['KERNEL_CLASS']) || isset($_ENV['KERNEL_CLASS'])) {
```

Comment:
This should probably be fixed in 2.7.

Reply:
Removed changes regarding to `KERNEL_DIR` var only, so `KERNEL_CLASS` was introduced in `3.3`.

Reply:
See https://github.com/symfony/symfony/pull/24115